app/views/api/asciicasts/show.html.slim

```diff
@@ -19,6 +19,14 @@ javascript:
       if (e.data == 'asciicast:play' || e.data[0] == 'asciicast:play') {
         player.play();
       }
+
+      // reveal.js support
+
+      if (e.data == 'slide:start') {
```

Comment:
Why not simply make it conditional on asciinema's, rather than reveal,js', `data-autoplay`?